web/src/stores/user.ts

```diff
@@ -7,6 +7,25 @@ export const VISIBLE_FOR_NONE = 0;
 export const VISIBLE_FOR_ALL = 1;
 export const VISIBLE_FOR_TEAM = 2;
 
+function getDefaultState(): User.State {
+  return {
+    userId: generateGUID(),
+    authToken: generateToken(),
+    email: null,
+    sendEmails: false,
+    hasDownloaded: false,
+    privacyAgreed: false,
+    recordTally: 0,
+    validateTally: 0,
+    isSubscribedToMailingList: false,
+
+    userClients: [],
+    isFetchingAccount: true,
+    account: null,
+  };
+}
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
"Fixing" this by disabling the eslint rule because the alternative is a lot of effort for something that does not give us any immediate value. Perhaps we can revisit this in the future